lib/launchers/amz.rb

```diff
@@ -52,6 +52,8 @@ def initialize(access_key: nil, secret_key: nil, service_name: nil, region: nil)
         )
       }) )
       Aws.config.update( config[:config_opts].merge({region: region})) if region
+
+      @account_id = awscred["aws_account_id"]
```

Comment:
please remove this line, the ID should be obtained within the method, not at initialization time

Reply:
Also it is not correct to put account id inside the awscred file. It is not standard part AFAIK.

Reply:
It is not standard but it doesn't break anything. Moving the id attribute somewhere else would make more mess IMHO.

Reply:
btw, where should I move it?

Reply:
The awscred file is standard file used by aws official cli. Modifying standard files is more mess. You can't reuse existing AWS accounts in config.yaml.

Reply:
Also most likely we can figure that value out from API calls. Do we know any OCM dev that can tell use why is this value needed and how exactly do they use it?

Reply:
I have removed it